services/cache/lru_test.go

```diff
@@ -122,10 +122,7 @@ func TestLRUMarshalUnMarshal(t *testing.T) {
 	var value2 map[string]interface{}
 	err = l.Get("test", &value2)
 	require.Nil(t, err)
-
-	v1, ok := value2["key1"].(int64)
```

Comment:
This was something which got changed. The library now returns int8 instead of int64. I checked the code. There's nowhere we are explicitly casting to `.(int64)` from a cached value. But it might be good to re-check this.